### PR TITLE
Add module for managing webhook payload templates

### PR DESCRIPTION
**Pull request by joshuajcoronado**
This module allows users to [manage webhook payload templates](https://developer.cisco.com/meraki/api-latest/#!delete-network-webhooks-payload-template) using the /networks/{networkId}/webhooks/payloadTemplates/ API.

I am unable to run integration tests, so I can't verify that I got every base covered.

```- name: Create payload template
  meraki_webhook_payload_template:
    auth_key: abc12345
    org_name: YourOrg
    state: query
    name: TestTemplate
    body: Testbody
    headers:
        - name: testheader
          template: testheadertemplate
  delegate_to: localhost
```

### CODE REVIEW

**Comment by joshuajcoronado** [comment]
Was able to get integration tests running locally, this is ready for review

**Review by kbreit — CHANGES_REQUESTED** [review]

**Comment by kbreit** [comment]
I had a few minor changes but tests pass and with the changes, I think I'm good merging this. Thank you!

**Comment by joshuajcoronado** [comment]
will update real quick

**Comment by joshuajcoronado** [comment]
@kbreit just added check mode tests, and update some of the functions

File: plugins/modules/meraki_webhook_payload_template.py [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+            options=dict(
+                name=dict(type="str"),
+                template=dict(type="str"),
+            ),
+        ),
+    )
+
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
```
[the remark this answers is not in the document]
Comment (kbreit) [reply plugins/modules/meraki_webhook_payload_template.py:233]: I didn't see it does actually support check mode.

**Review by kbreit — APPROVED** [review]

**Comment by kbreit** [comment]
Thank you for your contribution. I'm merging this PR.

**Pull request merged**